nexus/tests/integration_tests/projects.rs

```diff
@@ -94,4 +97,54 @@ async fn test_projects(cptestctx: &ControlPlaneTestContext) {
     .all_items;
     assert_eq!(projects.len(), 1);
     assert_eq!(projects[0].identity.name, p1_name);
+
+    // Unauthenticated and unauthorized users should not be able to delete or
```

Comment:
I realize all these manual tests for unauthenticated/unauthorized cases are getting long in the tooth.  I hope to replace these with an automated test that hits all endpoints unauthenticated and (separately) authenticated but unauthorized.  But I still want to test this stuff in the meantime.